lib/core/versions/latest/models/PublicKeyModel.ts

```diff
@@ -1,11 +1,11 @@
-import PublicKeyPurpose from '../PublicKeyPurpose';
+import VerificationRelationship from '../VerificationRelationship';
 
 /**
  * Data model representing a public key in the 'publicKey' array in patches.
  */
 export default interface PublicKeyModel {
   id: string;
   type: string;
-  jwk: any;
```

Comment:
Pushing back on this change as this is what is currently in spec and unrelate to casing change.

Reply:
https://w3c.github.io/did-core/#key-types-and-formats

https://tools.ietf.org/html/rfc7517

JWK could be a private or symmetric key, luckily we use the term `privateKey` to refer to JWKs that have a `d` values in sidetree code base today....would still prefer the term `privateKeyJwk` instead... 

might as well match did core, also makes JSON production easier, 1 less internal property that needs to be mapped to did core external representation.

Reply:
I buy the argument about being explicit and differentiate from "private key" for this one in principle, but have to push back because this is not part of the agreed upon change and taking this would mean more changes on clients/services and also backwards compat code changes in ref-imp.

Reply:
I disagree, we agreed to implement breaking changes for clients, all properties under `public_keys` are now possible to align with did core, and provide a better development experience for everyone who relies on the sidetree protocol, Transmute, SecureKey, Microsoft, ConsenSys, and others... The working group should determine the design aesthetics of the sidetree protocol, not a specific did method consuming client team.

Reply:
I'd prefer to see all changes like this go in now and have them set in stone. In this case publicKeyJwk is preferred for consistency to the did-core spec so that it adheres to the principle of least astonishment.

Reply:
Consensus to use `publicKeyJwk`.